admin-manual/installation-setup/installation/installation.rst

```diff
@@ -20,19 +20,19 @@ Archivematica is not a single application - dozens of different components and
 tools are required for a fully working installation. As a result, there are many
 possible deployment configurations.
 
-These instructions are designed to get you up and running as quickly as
-possible, even if you are not familiar the various tools and applications that
+The following table lists all help topics on installing Archivematica for various intended uses and host environments. The related document links are designed to get you up and running as quickly as
+possible, even if you are not familiar with the various tools and applications that
 are bundled into Archivematica. Experience with the Linux command line is
 helpful, and to support a running production system it should be considered a
 requirement.
 
 If you need assistance or clarification regarding the installation instructions,
 the `Archivematica google group`_ is a good place ask questions.
 
-.. note:: For testing purposes, you may find it easier to install on a virtual
-   machine using Vagrant. See the :ref:`Quick Start Guide <quick-start-install>`
+.. csv-table::
+   :file: _csv/install-landing.csv
```

Comment:
This is really cool, I had no idea you can do that!
I really like the way it's presented!

Reply:
Thanks! I stumbled across it myself while looking for an alternative to using the list-table directive. (*shudder*)